tests/test_configuration_set.py

```diff
@@ -101,3 +104,96 @@ def test_configurations_load_xyz():
+            'H  -0.03367  -1.48400  -0.92261   -0.15960    0.52124   -0.26696',
+            'O  -0.64023   1.65497  -1.71141   -1.17861    2.90807    0.06166',
+            'C   0.83922   0.14416  -2.83825   -1.44421   -0.12439    0.41955',
+            'H   1.17303   1.03489  -3.32904    0.44566    0.30180   -0.40030',
+            'H   0.31973  -0.52220  -3.55093    0.22722    0.31219    0.27423',
+            'H   1.62607  -0.41397  -2.36310    0.46565   -0.16544    0.01173',
+            '18',
+            'Lattice="18.000000 0.000000 0.000000 0.000000 18.000000 0.000000 0.000000 0.000000 18.000000" '
+            'energy=-11581.02323085 Properties=species:S:1:pos:R:3:forces:R:3',
+            'C  -0.14388   0.58514   1.72503   -0.26651    0.13421    1.07021',
+            'C   0.04181  -0.85624   1.27377   -1.15828    0.33661    0.32617',
+            'C   1.30547  -1.19982   1.43121    2.41677    0.72934    0.38652',
+            'H  -0.77023  -1.47035   0.85124    0.66526    0.54979    0.49939',
+            'C   2.09489  -0.00345   1.95796   -1.16435   -0.35668    0.47563',
+            'H   1.80942  -2.13489   1.19406   -0.38060    0.13724    0.00064',
+            'C   1.25045   1.00558   2.20650   -1.11760   -2.07214   -1.44189',
+            'H   3.11668   0.10081   2.28762    0.28187   -0.61557   -0.35632',
+            'H   1.56030   1.92262   2.59835    0.05989    1.31003    0.53325',
+            'H  -0.95605   0.73278   2.48750    0.81396   -0.33790   -0.40133',
+            'H  -0.50535   1.20087   0.91714    0.01837    0.30916   -0.51288',
+            'C  -0.86571   1.19876  -2.16169   -2.75334    2.99743    0.65360',
+            'C  -1.83154   2.15751  -1.85043    1.55884   -2.59191   -1.02838',
+            'H  -2.66091   1.87770  -1.19443    0.59674    0.16747    0.12963',
+            'H  -1.91392   3.10727  -2.37756    0.24926   -0.27902    0.12722',
+            'C  -0.83439  -0.11351  -1.50115   -0.63239    0.41365   -0.46196',
+            'H  -0.18981   1.36454  -2.96508    0.72714    0.38914   -0.29681',
+            'O  -1.79851  -0.44558  -0.83933    0.63063   -0.60796   -0.20044',
+            sep='\n',
+            file=xyz_file,
+        )
+
+    configs.load_xyz(
+        'tmp.xyz', charge=0, mult=1, load_energies=True, load_forces=True
+    )
+
+    assert len(configs) == 2
+
+    # check loading properties line
+    energies = [-11580.70167936, -11581.02323085]
+    num_atoms = [22, 18]
+    box_sizes = [(20, 20, 20), (18, 18, 18)]
+    for i in range(2):
+        config = configs[i]
+        assert config.box == Box(box_sizes[i])
+        assert config.charge == 0
+        assert config.mult == 1
+        assert config.energy.true == energies[i]
+        assert len(config.atoms) == num_atoms[i]
+        assert len(config.coordinates) == num_atoms[i]
+        assert len(config.forces.true) > 0
+
+    # check config set force and energy loading
+    assert len(configs.true_forces) > 0
```

Comment:
This part can go to the test of ConfigurationsSet and test exact values, not just the length